src/application.rs

```diff
@@ -516,6 +613,48 @@ impl Application {
                     self.ui.show_ip_dialog(iface_data);
                 }
             }
+            UiActions::ChangeServer => {
+                let is_onboarded = self.model.borrow().node_status.is_onboarded();
+
+                if is_onboarded {
```

Comment:
you're not using the variable here, why not

```rust
if self.model.borrow().node_status.is_onboarded() {
// ...
}

```

Reply:
no specific reason really. I can change it

Reply:
no reason :) i'll change